### ISSUE TITLE
gcc-arm-none-eabi Install instructions out of date for Ubuntu 18.04?

### ISSUE THREAD

**Issue by U007D** [rust-embedded/embedonomicon#40]
16.04 instructions did not work for me and some search results suggested this has changed for 18.04.
Not sure if this is the best way now, but these instructions worked: #38

**Comment by therealprof** [comment]
Thanks for the report.

Where did you find the gcc install instructions? gcc is not really required anymore so if there're some leftovers we should clean them up.

Anyway, 18.04 should has the usual suspects available: https://packages.ubuntu.com/bionic/gcc-arm-none-eabi

**Comment by U007D** [comment]
I just tried to follow the 16.04 instructions [here](https://github.com/rust-embedded/embedonomicon/blob/master/src/preface.md#ubuntu-1604).

**Comment by U007D** [comment]
True--I used it to get `gdb`, but as you pointed out in the code review comment `gdb-multiarch` makes a lot more sense, I agree.

**Comment by therealprof** [comment]
As mentioned in #38 there might be still some usefulness to having a section explaining how to install a generic toolchain bundle from ARM. If you would be so kind to move the instructions to a new section and update the Ubuntu 18.04 for `gdb-multiarch` (probably after verifying that it actually works ;) ), that'd be really awesome.

**Comment by U007D** [comment]
Done.  Please feel free to close this issue if the new edits are good to go.
